### PR TITLE
[Mobile] Show status-go version on separate line in About.

### LINKED ISSUE #7231: Version number is not fully visible in About screen
### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: Version number is not fully visible in About screen
screenshot_20190109-112517

#### Expected behavior
[comment]: # (Describe what you expected to happen.)
Version is fully visible. For example, it goes to second line if it does not fit 1 line
#### Actual behavior
[comment]: # (Describe what actually happened.)
End of the version string "Version 0.9.33(2019010910); node 0.17.11-bet..."  is not visible.

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)

- Open Status on iPhone 5s or Galaxy S6
-  Profile -> About
- Check if string for version number is fully visible

### Additional Information
[comment]: # (Please do your best to fill this out.)
* Status version: release build 1 for 0.9.33
* Operating System: Android and iOS

### PR DESCRIPTION
**Pull request by mandrigin**
fixes #7231

### New look (iPhone 5s):
iphone se - 12 1 2019-01-09 16-41-59

#### Platforms (optional)
- Android
- iOS
- macOS
- Linux
- Windows

#### Areas that maybe impacted (optional)
**Functional**
- about

status: ready

### CODE REVIEW

**Review by rasom — APPROVED** [review]

File: src/status_im/ui/screens/about_app/views.cljs [hunk 1]
```diff
@@ -8,7 +8,8 @@
             [re-frame.core :as re-frame]))
 
 (views/defview about-app []
-  (views/letsubs [version [:get-app-version]]
+  (views/letsubs [app-version [:get-app-short-version]
```
Comment (flexsurfer) [inline src/status_im/ui/screens/about_app/views.cljs:11]: there is no need to use subscription here because it's a constant

**Review by flexsurfer — APPROVED** [review]

**Review by vkjr — APPROVED** [review]

File: src/status_im/ui/screens/about_app/views.cljs (hunk 1 above)
Comment (mandrigin) [reply src/status_im/ui/screens/about_app/views.cljs:11]: Yeah, i just tried to keep it consistent :)

File: src/status_im/ui/screens/profile/subs.cljs [hunk 2]
```diff
@@ -4,6 +4,13 @@
             [status-im.utils.build :as build]
             [status-im.utils.platform :as platform]))
 
+(defn- node-version [{:keys [web3-node-version]}]
+  (str "status-go v" (or web3-node-version "N/A") ""))
```
Comment (pedropombeiro) [inline src/status_im/ui/screens/profile/subs.cljs:8]: What is the purpose of the empty string suffix `""`?

Comment (mandrigin) [reply src/status_im/ui/screens/profile/subs.cljs:8]: I have no clue, I just kept it from the original code :(

**Comment by churik** [comment]
Tested on IOS 11, IPhone 7 and Android 7.1.0(Samsung Galaxy S7, Genymotion emulator)

**Pull request merged**